tools/preflight/preflight_test.go

```diff
@@ -117,8 +117,94 @@ func preflightTestCases(serverVersion string) {
 }
 
 var _ = Context("Preflight Unit Tests", func() {
```

Comment:
need test-cases for all major funcs written for each preflight check.. add tests for missing ones

Reply:
Major tests such DNS resolution, volume snapshot check and cleanup with/withhout uid will added in separate ticket